openpype/hosts/resolve/api/workio.py

```diff
@@ -72,15 +75,19 @@ def open_file(filepath):
         return False
     return True
 
+
 def current_file():
     pm = get_project_manager()
     current_dir = os.getenv("AVALON_WORKDIR")
     project = pm.GetCurrentProject()
     name = project.GetName()
     fname = name + exported_projet_ext
     current_file = os.path.join(current_dir, fname)
+
+    # check if file exists
```

Comment:
Comment is incorrect - there is no checking done here whether the file exists at all.